tests/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# TODO(Alessandro): This library is duplicated, since we already have one
+#                   copy in the /external folder. They are not at the same
+#                   version and so for now we have to include this too.
```

Comment:
see earlier point about TODO comments.  (Same for any others in the PR, I'll stop commenting on it now)